src/pages/contrato.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import Link from 'next/link';
+import Head from '../components/head';
+import linebreaksbr from '../lib/linebreaksbr';
+import { getContract } from '../lib/api';
+
+class Contratos extends React.Component {
+  static async getInitialProps({ query }) {
+    let slug = query.slug;
+    let data = await getContract({ slug });
+    return data;
+  }
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      contract: props
+    };
+  }
+
+  render() {
+    return (
+      <div>
+        <Head title="Contratos" />
+        <div className="flex self-start justify-center">
+          <div className="w-full mb-4">
+            <div className="flex flex-wrap">
+              <div className="w-full sm:flex-1 px-4 py-2 m-2 border-none border-r sm:border-solid sm:border-grey-light text-lg text-grey-darkest">
+                <h2 className="mb-2">{this.state.contract.number}</h2>
+
+                {this.state.contract.parent ? (
+                  <>
+                    <p className="font-bold">Enmienda</p>
+                    <p className="mb-2">
+                      {this.state.contract.amendment} -
+                      <Link
+                        href={`/contratos?slug=${
+                          this.state.contract.parent.slug
+                        }`}>
+                        <a className="text-grey-darkest hover:text-black">
+                          Contrato original
+                        </a>
+                      </Link>
+                    </p>
+                  </>
+                ) : null}
+
+                <p className="font-bold">Cuantía</p>
+                <p className="mb-2">${this.state.contract.amount_to_pay}</p>
+
+                <p className="font-bold">Categoría de servicio</p>
+                <p className="mb-2">{this.state.contract.service.group.name}</p>
+
+                <p className="font-bold">Tipo de servicio</p>
+                <p className="mb-2">{this.state.contract.service.name}</p>
+
+                <p className="font-bold">Entidad</p>
+                <p className="mb-2">
+                  <Link href={`/entidad/${this.state.contract.entity.slug}`}>
+                    <a className="text-grey-darkest hover:text-black">
+                      {this.state.contract.entity.name}
+                    </a>
+                  </Link>
+                </p>
+
+                <p className="font-bold">Otorgado</p>
+                <p className="mb-2">{this.state.contract.date_of_grant}</p>
```

Comment:
use camelCase? In this case, this might be coming from the api in which case a snake case to camel case conversion could be implemented on the response level.